<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>630</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>728</width>
    <height>630</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>728</width>
    <height>630</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>13</pointsize>
    <italic>false</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Manual enter&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>60</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Preview</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Item code</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <italic>false</italic>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>21</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>28</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;/&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <italic>false</italic>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>SKU parameters</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="title">
     <string>Letter</string>
    </property>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>86</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>13</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <item>
      <property name="text">
       <string>A</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>B</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>C</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>D</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>E</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>G</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>H</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>I</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>J</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>K</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>L</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>M</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>N</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>O</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>P</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Q</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>R</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>S</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>T</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>U</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>V</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>W</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>X</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Y</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Z</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>411</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="title">
     <string>Length</string>
    </property>
    <widget class="QPlainTextEdit" name="plainTextEdit_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>13</pointsize>
       <italic>false</italic>
      </font>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_4">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>50</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>13</pointsize>
       <italic>false</italic>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>13</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="text">
      <string>Length or Size 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>13</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="text">
      <string>Size 2</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>158</width>
      <height>254</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="title">
     <string>Karat</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <widget class="QRadioButton" name="radioButton_4">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Platinum</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="radioButton_2">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>14K</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QRadioButton" name="radioButton_5">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Silver</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="radioButton">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>10K</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QRadioButton" name="radioButton_3">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Steel</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QRadioButton" name="radioButton_6">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>18K</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>221</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="title">
     <string>Color</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="9" column="0">
      <widget class="QRadioButton" name="radioButton_12">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Three tones</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QRadioButton" name="radioButton_14">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Black</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QRadioButton" name="radioButton_13">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Yellow/Rose</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QRadioButton" name="radioButton_9">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>White/Yellow</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QRadioButton" name="radioButton_8">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>White</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QRadioButton" name="radioButton_10">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Rose</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QRadioButton" name="radioButton_11">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>White/Rose</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="radioButton_7">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Yellow</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>360</y>
      <width>261</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkBox">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>SKU 6 digits max</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="checkBox_2">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Cut do not pull</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="checkBox_4">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>LAB</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Manual enter SKU&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="text">
     <string> Manual</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>26</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(143, 240, 164);</string>
    </property>
    <property name="text">
     <string>PRINT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_qr">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>90</y>
      <width>264</width>
      <height>264</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>500</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QRadioButton" name="radioButton_15">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>DYMO</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="radioButton_16">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>13</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Zebra</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>groupBox_3</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>plainTextEdit</zorder>
   <zorder>label_3</zorder>
   <zorder>plainTextEdit_2</zorder>
   <zorder>label_4</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>groupBox_5</zorder>
   <zorder>checkBox_3</zorder>
   <zorder>pushButton</zorder>
   <zorder>label_qr</zorder>
   <zorder>groupBox_6</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>728</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
